src/pudl/metadata/fields.py

```diff
@@ -1139,6 +1139,7 @@
             "A 2-3 letter code indicating the energy source (e.g. fuel type) "
             "associated with the record."
         ),
+        # Should this have an enum reference to the core_eia__codes_energy_sources table??
     },
     "energy_source_code_num": {
         "type": "string",
```

Comment:
I noticed that none of the `energy_source_code` fields (ex: `energy_source_code_num`, `energy_source_code_1`, `energy_source_code`) have `enums` established. There is a coding table for energy source code: `core_eia__codes_energy_sources` -- should we be referencing that in the field enum? 

Will values get encoded if there is no enum established in the field?

Reply:
ahhh ok thanks for the clarification. Isn't the whole point of the encoding process to weed out erroneous values? Is it not duplicative to have the encoding step in the transformer and the enums defined in the field metadata?

Reply:
Maybe I'm not understanding what you're saying or thinking here, but this field does not indicate a fuel type. It indicates which of the several available `energy_source_code` slots some other column's fuel type was reported in, so we can identify which one is most important (N=1), second most important (N=2) etc.